Gruntfile.js

```diff
@@ -9,8 +9,8 @@ module.exports = function (grunt) {
           sourcemap: true,
           includePaths: [
             'govuk_modules/govuk_template/assets/stylesheets',
-            'hmrc_modules/govuk_elements/govuk/public/sass',
-            'hmrc_modules/govuk_frontend_toolkit/stylesheets',
+            'hmrc_modules/assets-frontend/govuk_elements/govuk/public/sass',
+            'hmrc_modules/assets-frontend/govuk_frontend_toolkit/stylesheets',
```

Comment:
I wonder if these might be even more confusing? could all the govuk directories be copied into a single `govuk` directory inside `assets_frontend`?

```
.
└── assets-frontend
    ├── govuk
    │   ├── govuk_elements
    │   └── govuk_frontend_toolkit
    ├── images
    └── scss
```